tests/unit/ManualBusinessInfo.spec.ts

```diff
@@ -35,29 +38,40 @@ describe('Manual BusinessInfo component', () => {
       homeIncorporationDate: '',
       homeLegalName: '',
       mode: 'MANUAL'
-    }
-    const getters = {
-      isContinuationInAffidavitRequired: () => true,
-      getExistingBusinessInfo: () => businessMock
-    }
```

Comment:
Good idea but not needed as it was easy to initialize the store to match what was needed here.